src/gui/properties/trackerlist.cpp

```diff
@@ -524,13 +556,65 @@ void TrackerList::showTrackerListMenu(QPoint) {
   }
 }
 
-void TrackerList::loadSettings() {
-  if (!header()->restoreState(Preferences::instance()->getPropTrackerListState())) {
-    setColumnWidth(0, 30);
-    setColumnWidth(1, 300);
-  }
+void TrackerList::loadSettings()
+{
+    header()->restoreState(Preferences::instance()->getPropTrackerListState());
+}
+
+void TrackerList::saveSettings() const
+{
+    Preferences::instance()->setPropTrackerListState(header()->saveState());
+}
+
+QStringList TrackerList::headerLabels()
+{
+    QStringList header {
```

Comment:
or remove this method and put the following to top of the file instead:
```c++
const QStringList headerLabels = {
   ...
}
```
and use `headerLabels[i]` below

@thalieht your choice